### DISCUSSION TITLE
tqdm code no longer working in 03_pytorch_computer_vision

### DISCUSSION THREAD

**Discussion by git-tyl** [mrdbourke/pytorch-deep-learning#111] [Q&A] [answered]
Hi,

A couple of days back the tqdm code in my colab notebook was showing the progress bar. However, now it doesn't show it.

Was there a change in google codelab to stop it from working?

I have tried loading the master branch 03_pytorch_computer_vision jupiter notebook into colab but it fails as well.

*update I've just tried it on vscode on my mac using the m1 chip seems there is a problem with ipywidgets https://stackoverflow.com/questions/73407189/vs-code-interactive-is-not-rendering-tqdm-notebook-properly without moving ipywidgets back to 7.7.1 it won't work in the mac. maybe thats what's happening in colab? ipywidgets needs to be downgraded? I still haven't figured out how to get it to work in colab, but maybe I'll stick to using vscode for a while, local running seems nicer than colab.

**Comment by mrdbourke** [comment, marked as answer]
Hey @git-tyl,

You're right, I'm getting the same issue on Google Colab too.

However, it's still working for me locally as well.

I'm not sure on a fix yet but I will update this when I find one.

**Reply by mrdbourke** [reply]
Hi @git-tyl,

I'm now getting tqdm progress bars back in Google Colab as of today: 

<img width="2032" alt="Screen Shot 2022-08-26 at 3 04 07 pm" src="https://user-images.githubusercontent.com/16750345/186826062-153c7ea2-aa60-45ef-adde-027989e8930b.png">
